python-checks/src/main/java/org/sonar/python/checks/tests/ImplicitlySkippedTestCheck.java

```diff
@@ -92,6 +101,30 @@ private static ReturnStatement getReturnStatementFromFirstIfStatement(FunctionDe
     return null;
   }
 
+  private static boolean hasAnySkipStatement(List<Expression> expressions) {
+    return expressions.stream()
+      .filter(expression -> expression.is(Tree.Kind.CALL_EXPR))
+      .map(CallExpression.class::cast)
+      .anyMatch(callExpr -> isPytestSkip(callExpr) || isUnittestSkip(callExpr));
+  }
+
+  private static boolean isPytestSkip(CallExpression callExpression) {
+    return Optional.of(callExpression).stream()
+      .filter(callExpr -> callExpr.callee().is(Tree.Kind.QUALIFIED_EXPR))
+      .map(callExpr -> (QualifiedExpression) callExpr.callee())
```

Comment:
This part is identical in `isPytestSkip` and `isUnittestSkip`. Maybe you can move it to `hasAnySkipStatement`.